Revit_Engine/Create/Config/RevitPullConfig.cs

```diff
@@ -46,6 +46,20 @@ public static RevitPullConfig RevitPullConfig(Discipline discipline = Discipline
 
         /***************************************************/
 
+        [Description("Creates a pull action-specific configuration used for adapter interaction with Revit.")]
+        [InputFromProperty("discipline")]
+        [InputFromProperty("includeClosedWorksets")]
+        [InputFromProperty("geometryConfig")]
+        [InputFromProperty("representationConfig")]
+        [InputFromProperty("includeNestedElements")]
+        [Output("revitPullConfig")]
+        public static RevitPullConfig RevitPullConfig(Discipline discipline = Discipline.Undefined, bool includeClosedWorksets = false, PullGeometryConfig geometryConfig = null, PullRepresentationConfig representationConfig = null, bool includeNestedElements = true)
```

Comment:
Please deprecate the previous version of the method (possibly `PreviousVersionAttribute` could be used, too).